src/core/Akka/Util/Option.cs

```diff
@@ -77,5 +89,15 @@ public override int GetHashCode()
 
         /// <inheritdoc/>
         public override string ToString() => HasValue ? $"Some<{Value}>" : "None";
+
+        /// <summary>
+        /// Applies specified action to this option, if there is a value
+        /// </summary>
+        /// <param name="action"></param>
+        public void ForEach(Action<T> action)
```

Comment:
I'd rename this to be more idiomatic in C#, rather than borrow the Scala API

Reply:
Oh yes, I spent some small time thinking about more idiomatic name, but nothing got to my mind and I left it as it is... Renamed it to `OnSuccess` now - do you think it is more intuitive/idiomatic?

Reply:
Yep, that's better.